### ISSUE TITLE
How to run LR finder?

### ISSUE THREAD

**Issue by MichaelMonashev** [rwightman/efficientdet-pytorch#119]
https://github.com/davidtvs/pytorch-lr-finder needs to specify criterion to find best LR:
```
lr_finder = LRFinder(model, optimizer, criterion, device)
lr_finder.range_test(train_loader, end_lr=end_lr, num_iter=30, step_mode="exp", accumulation_steps=accumulation_steps)
lr_finder.reset()
```
How to pass it?

**Comment by rwightman** [comment]
Beyond the scope of this repository. The bench contains criterion and outputs the losses in output as dict keys, you'd likely have to modify the finder to access the loss that way
